test/formulas.txt

```diff
@@ -89242,3 +89242,4 @@
 "SUM(I56+F11)"
 "SUM(B3:B10)"
 "SUM(B3:B10)"
+"ARRAY(“string1”, "string2")"
```

Comment:
I don't think this is the right place to add a unit test because there's no expectedValue. Sync with @marhong2 on how to add a unit test to fast formula parser.